shared-libs/cht-datasource/src/person.ts

```diff
@@ -108,6 +91,50 @@ export namespace v1 {
    * @param context the current data context
    * @returns a function for retrieving a paged array of people
    * @throws Error if a data context is not provided
+   * @see {@link getAll} which provides the same data, but without having to manually account for paging
    */
-  export const getPage = getPeople(Local.Person.v1.getPage, Remote.Person.v1.getPage);
+  export const getPage = (
+    context: DataContext
+  ): ({ personType, limit, skip }: GetPageParams) => Promise<Person[]> => {
```

Comment:
```suggestion
  ): typeof curriedFn => {
```

Got to do it this way to get the docs from `curriedFn` to show up in the site... :sweat: